### PR TITLE
Add support for zAdd options like XX|NX and so on

### PR DESCRIPTION
**Pull request by lovelock**
https://github.com/phpredis/phpredis/issues/760
phpredis now support the options but the stubs not

### CODE REVIEW

File: redis/Redis.php [hunk 1]
```diff
@@ -2306,30 +2306,45 @@ public function brpoplpush( $srcKey, $dstKey, $timeout ) {}
...
+     * @param   string  $value2  Optional value
+     * @param   float   $scoreN  Optional score
+     * @param   string  $valueN  Optional value
      * @return  int     Number of values added
      * @link    https://redis.io/commands/zadd
      * @example
      * <pre>
      * <pre>
-     * $redis->zAdd('z', 1, 'v2', 2, 'v2', 3, 'v3', 4, 'v4' );  // int(2)
+     * $redis->('z', 1, 'v1', 2, 'v2', 3, 'v3', 4, 'v4' );  // int(2)
```
Comment (King2500) [inline redis/Redis.php]: `->(` is that correct syntax?

Comment (lovelock) [reply redis/Redis.php]: Ahh thanks, I've fixed a typo and introduced another one.

File: redis/Redis.php [hunk 2]
```diff
@@ -2306,30 +2306,45 @@ public function brpoplpush( $srcKey, $dstKey, $timeout ) {}
...
      * Adds the specified member with a given score to the sorted set stored at key.
      *
-     * @param   string  $key    Required key
-     * @param   float   $score1 Required score
-     * @param   string  $value1 Required value
-     * @param   float   $score2 Optional score
-     * @param   string  $value2 Optional value
-     * @param   float   $scoreN Optional score
-     * @param   string  $valueN Optional value
+     * @param   string  $key     Required key
```
Comment (pestretsov) [inline redis/Redis.php]: Please, revert the formatting, so the only change in `@params` block is the added `@param array $options Options if needed`. We [do this to preserve git history](https://github.com/JetBrains/phpstorm-stubs/blob/master/CONTRIBUTING.md#code-style).

**Comment by lovelock** [comment]
Thanks for your review and the format has been reverted.

**Comment by pestretsov** [comment]
Thank you!

**Pull request merged**